modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -143,6 +143,10 @@ export default function _JournalPortlet({
 		publishingLock.unlock();
 		console.error(error);
 
+		if (error.error?.statusCode) {
+			showAlert(error.error.message);
+		}
```

Comment:
@IstvanD the @liferay-echo team needs to review this change

Reply:
@carolmariaabb I have sent it to the Echo Team as well:
https://github.com/liferay-echo/liferay-portal/pull/14050